Servers/controllers/project.ctrl.ts

```
@@ -149,18 +149,12 @@ export async function createProject(req: Request, res: Response): Promise<any> {
-    const createdProject = await createNewProjectQuery(
-      newProject,
-      newProject.members
-    );
-    const assessments: Object = await createNewAssessmentQuery(
-      {
-        project_id: createdProject.id,
-      },
-      newProject.enable_ai_data_insertion
-    );
+    const createdProject = await createNewProjectQuery(newProject, newProject.members);
+    const assessments: Object = await createNewAssessmentQuery({
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)